src/openrct2/world/map.h

```diff
@@ -352,6 +354,8 @@ extern sint16 gMapSize;
 extern sint16 gMapSizeMaxXY;
 extern sint16 gMapBaseZ;
 
+extern sint16 gMapCtrlPressed;
```

Comment:
This should be a bool, not a sint16.

Reply:
I believe other gMapXXXXPressed are also sint16.

Didnt want to touch them in this feature (same as before, this would mean some values are bool, some sint16 - for different keys...).

This should be corrected with my dragging feature as it refactores bigger chunk of the code.

Reply:
It's new, just make it bool, unless it means it's incompatible.